docs/Mirrors.md

```diff
@@ -70,7 +70,13 @@ You can create a public AlmaLinux mirror in 6 easy steps:
     subnets:
       - <network mask>
     # That parameter can specify ASN which is maintained by a mirror is located in cloud
-    asn: <ASN number, see https://en.wikipedia.org/wiki/Autonomous_system_(Internet)>
+    # It can be string or list of strings:
```

Comment:
Maybe I'm wrong, but I'd say that, since ASN is already indicating that we're talking about numbers (as the N of ASN stands), we can indicate that the value can be either an integer or a list of integers, and that can be represented in different ways, such as:

```
asn: 54321
asn: [9876, 54321]
asn:
    - 9876
    - 54321
```
Is that accurate?

Reply:
Yep, you are right. I thought about that the service converts all of them to strings for comparing when writing this hint. Fixed.